pkgs/applications/version-management/git-bug/default.nix

```
@@ -1,9 +1,9 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
 
 buildGoModule rec {
   pname = "git-bug";
-  version = "0.8.0"; # the `rev` below pins the version of the source to get
-  rev = "v0.8.0";
+  version = "0.8.0"; # explicit commit hash is passed in ldflags, needed for version info
+  rev = "a3fa445a9c76631c4cd16f93e1c1c68a954adef7";
```

```suggestion
  version = "0.8.0";
```
we don't need that and makes auto updates easier

---

There is a `git-bug version --commit` command which normally outputs the whole hash, but I guess that's a small breakage if someone is relying on that to be a hash and not any ref.